idunn/blocks/stars.py

```diff
@@ -0,0 +1,56 @@
+from enum import Enum
+from typing import List, Literal, Optional
+from pydantic import BaseModel
+from .base import BaseBlock
+
+
+class StarsAvailable(Enum):
+    YES = "yes"
+    NO = "no"
+
+    @classmethod
+    def from_bool(cls, val):
+        if val:
+            return cls.YES
+
+        return cls.NO
+
+
+class StarsKind(Enum):
+    LODGING = "lodging"
+    RESTAURANT = "restaurant"
+
+
+class StarsDetails(BaseModel):
+    has_stars: StarsAvailable
+    nb_stars: Optional[float]
+    kind: StarsKind
+
+
+class StarsBlock(BaseBlock):
+    type: Literal["stars"] = "stars"
+    ratings: List[StarsDetails]
+
+    @classmethod
+    def from_es(cls, place, lang):
+        ratings = [
+            StarsDetails(
+                has_stars=(
+                    StarsAvailable.from_bool(stars)
+                    if isinstance(stars, bool)
+                    else StarsAvailable.YES
+                ),
+                nb_stars=stars if isinstance(stars, float) else None,
+                kind=kind,
+            )
+            for kind, stars in [
+                (StarsKind.LODGING, place.get_lodging_stars()),
+                (StarsKind.RESTAURANT, place.get_restaurant_stars()),
+            ]
```

Comment:
To help with the integration, perhaps we could keep only one of these ratings, depending on `place.get_class_name()` ?   
Then the client could just append the number of stars with the title of the poi class/subclass, without any redundant implementation on both sides, or risk of confusion.

cc @xem @bbecquet

Reply:
What I had in mind is that erdapfel would blindly display all items of the list, but maybe we risk displaying irrelevant data?